requirements.yml

```diff
@@ -4,9 +4,8 @@
 collections:
   - name: prometheus.prometheus
     version: 0.4.0
+  - name: grafana.grafana
```

Comment:
Can you add a fixed version here? It's better for maintenance purposes.

Reply:
ok I'm on it

Reply:
It's all good now